### ISSUE TITLE
Cannot reproduce the performance reported on AAAI camera ready version.

### ISSUE THREAD

**Issue by Zy-Zhang** [MCC-WH/Token#13]
Dear author, thanks so much for kindly providing the code. Meanwhile, your work is so great and interesting. However, I've tried to reproduce your work with the source code you provided but cannot obtain the performance even using all of the Google Landmark v2 clean instead of 80% of it. Also, I've gone through all of the issues reported here and found there's a big difference on the training parameters setting between the one you gave and log you showed [https://github.com/MCC-WH/Token/issues/7] in one of the issues. It seems you are using a batch size of 320 with initial learning rate of 0.01. Could you please kindly tell me which one is correct? I've tried to reproduce your performance based on the settings in the paper, but I failed. So could you tell me how exactly you obtain the performance? Thanks so much.

**Comment by MCC-WH** [comment]
Thank you for your interest in our paper. The learning rate in my code should change with batch size, and I think this may not be a major issue. I recently noticed a problem in my code in that the ResNet model does not import pre-trained parameters, did you change this? I am very sorry for not having time to fix it yet. The model I trained follows (https://github.com/filipradenovic/cnnimageretrieval-pytorch/blob/master/cirtorch/networks/imageretrievalnet.py) for model initialization.

**Comment by Zy-Zhang** [comment]
Thanks so much for replying me. I found that in your code, the real learning rate is set as ''lr x batchsize / 256'' in the warmup learning schedule part. Does it mean your real learning rate is set as 0.005 when you set it as 0.01 (since batchsize=128)? Also, I've noticed the model initialization issue in the code and I directly downloaded the pretrained model from Pytorch torchvision. Maybe this is another reason I cannot reproduce the performance. I will try to initialize the model with the one provided by filip.
